mobile/src/services/groups/groups.service.ts

```diff
@@ -0,0 +1,29 @@
+import { ApiPath, GroupsApiPath, HttpMethod } from '~/common/enums/enums';
```

Comment:
Filename and folder should be the same as a service name, rename them to to groups-api and groups-api.service.ts